lib/rt_rubocop_defaults/version.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+
 module RTRuboCopDefaults
-  VERSION = "1.1.0"
+  VERSION = "1.2.0.rc01"
```

Comment:
Would this be a major version update for `RTRuboCopDefaults`

Reply:
no by fixing the config/cop name we keep the same behavior

Reply:
Including a link to the diff in the Changelog might be helpful..